integration/watch.ts

```diff
@@ -1,86 +1,150 @@
 const chokidar = require('chokidar');
 const spawn = require('child_process').spawn;
+type FsEvent = 'add' | 'addDir' | 'change' | 'unlink' | 'unlinkDir';
 
-main()
+const colors = {
+  none: '',
+  reset: '\x1b[0m',
+  green: '\x1b[32m',
+  red: '\x1b[31m',
+  yellow: '\x1b[33m',
+  blue: '\x1b[34m',
+  magenta: '\x1b[35m',
+  cyan: '\x1b[36m',
+};
+
+main();
 
 function main() {
-    const yarn = process.platform === 'win32' ? 'yarn.cmd' : 'yarn'
+  const yarn = process.platform === 'win32' ? 'yarn.cmd' : 'yarn';
+  const usePolling = process.argv.includes('--polling');
+  const tests = process.argv.slice(2).filter((x) => !x.startsWith('-'));
+  const watchOptions = { ignoreInitial: true, usePolling };
+
+  if (process.argv.includes('-h') || process.argv.includes('--help')) showHelp();
+  showSettings(tests);
+
+  chokidar
+    .watch('integration/*/*.proto', watchOptions)
+    .on('all', integrationHandler(yarn, 'proto2bin'));
```

Comment:
Pretty minor/mostly just curious, but can we pass `'add', 'change'` here to `on` instead of passing `'all'` and later dropping any non-add/non-change?

Reply:
Unfortunately not... the api only permits literal strings that are one of ` 'all' | 'add' | 'addDir' | 'change' | 'unlink' | 'unlinkDir'`. Passing a comma delimited string breaks the listener.

But I did change the `srcHandler` to only listen to `change` instead of `all` and then filtering out the rest. There I needed only 'change'.